wgsl/index.bs

```diff
@@ -300,17 +300,24 @@ The <dfn dfn noexport>scalar</dfn> types are [=bool=], [=i32=], [=u32=], and [=f
 The <dfn dfn noexport>numeric scalar</dfn> types are [=i32=], [=u32=], and [=f32=].
 
 ### Vector Types ### {#vector-types}
+
+A <dfn noexport>vector</dfn> is a grouped sequence of 2, 3, or 4 [=scalar=] components.
+
 <table class='data'>
   <thead>
     <tr><th>Type<th>Description
   </thead>
   <tr><td>vec*N*<*T*><td>Vector of *N* elements of type *T*.
                           *N* must be in {2, 3, 4} and *T*
                           must be one of the [=scalar=] types.
-                          We say *T* is the component type of the vector
+                          We say *T* is the <dfn noexport>component type</dfn> of the vector.
 </table>
 
-A vector type is a <dfn dfn>numeric vector</dfn> type if its component type is a [=numeric scalar=].
+A vector is a <dfn dfn>numeric vector</dfn> if its component type is a [=numeric scalar=].
+
+A key use case of a vector is to express both a direction and a magnitude.
```

Comment:
I have mixed feelings about this. Shaders do a lot of color operations, and these happen to also be represented with vectors. It's arguable if the key use case is a direction versus just a color, for example. Maybe we can just reword this to be a weaker statement?

Reply:
That's good feedback.  I've expanded it into a list. PTAL.